content/chapitres/ci.adoc

```diff
@@ -88,37 +130,48 @@ décrite par un fichier YAML.
 * Chemin : `.github/workflows/<nom du workflow>.yml`
 * On peut avoir _plusieurs_ fichiers donc _plusieurs_ workflows
 
-== Concepts de Github Actions - Évènement
+[source,shell]
+----
+.github/workflows
+├── build-and-unit-test.yaml
+├── acceptance-test.yml
+└── deploy.yaml
```

Comment:
nit: Those examples would be multiple jobs of the same workflow, how about different job names?

```suggestion
├── ci-cd.yaml
├── bump-dependency.yml
└── nightly-tests.yaml
```